studio/src/aem/aem.js

```diff
@@ -301,10 +302,31 @@ class AEM {
 
     async pollUpdatedFragment(oldFragment) {
         let attempts = 0;
+
+        const isCollection = oldFragment.model?.path === COLLECTION_MODEL_PATH;
+        const oldDefaultChild = isCollection
+            ? oldFragment.fields?.find((f) => f.name === 'defaultchild')?.values?.[0]
+            : undefined;
+
         while (attempts < MAX_POLL_ATTEMPTS) {
             attempts++;
             const newFragment = await this.sites.cf.fragments.getById(oldFragment.id);
-            if (newFragment.etag !== oldFragment.etag) return newFragment;
+
+            if (!newFragment) {
+                await this.wait(POLL_TIMEOUT);
+                continue;
+            }
+
+            const newDefaultChild = isCollection
```

Comment:
what is the change here, is the etag of a collection is not changing when collection cards are updated?

Reply:
what is defaultChild?

Reply:
in general, it's an attribute that is set in a particular card of a collection. In the use case of simplified pricing express card, it's the card that gets expanded by default in mobile:
<img width="349" height="500" alt="Screenshot 2025-09-03 at 10 21 57 AM" src="https://github.com/user-attachments/assets/82bb942f-1ce1-4d49-9be9-4da28fe1b762" />
Author would like to set this attribute on any card of the collection.